include/nbl/builtin/hlsl/blur/common.hlsl

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#ifndef _NBL_BOX_BLUR_COMMONS_
+#define _NBL_BOX_BLUR_COMMONS_
```

Comment:
you should have the full namespace in the header define, see other HLSL header files

Reply:
actually you're missing a namespace for the whole thing, I guess it should be `nbl::hlsl::central_limit_blur`

the folder should be `central_limit_blur` instead of just `blur`